src/pages/Login/Login.js

```diff
@@ -1,8 +1,105 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
 import './Login.scss';
 
 function Login() {
-  return <div className="login" />;
+  const [inputValue, setInputValue] = useState({
+    email: '',
+    password: '',
+  });
+
+  const { email, password } = inputValue;
+
+  const [isActiveEmailBorder, setIsActiveEmailBorder] = useState(false);
+  const [isActivePasswordBorder, setIsActivePasswordBorder] = useState(false);
+
+  const handleInput = event => {
+    const { name, value } = event.target;
+    setInputValue({
+      ...inputValue,
+      [name]: value,
+    });
+  };
+
+  const isValidInput = () => {
+    return email.length >= 1 && password.length >= 1;
+  };
+
+  const handleButtonAction = () => {
+    return isValidInput() ? 'loginButtonAction' : 'loginButtonInaction';
+  };
+
+  const handleButtonValid = () => {
+    // 1.validation
+    // 2. if(pass) login request to backend else alert return
+    // 3. if(login success) go to main else alert invalid user
+
+    // login request
+    // if(success)
+    // go to main
+    // else
+    // alert
+
+    // go to main
+
+    if (!isValidInput()) {
+      alert('please write a password or email address');
+    }
+  };
+
+  // function
+  // function return
+
+  return (
+    <main className="login">
+      <div className="loginTop">
+        <h1 className="loginTitle">LOG IN YOUR USER ACCOUNT</h1>
+      </div>
+      <div className="loginMiddle">
+        <form className="loginInput">
+          <div className="emailInput">
+            <div className="loginInputMassage">Email address *</div>
+            <input
+              name="email"
+              className={isActiveEmailBorder ? 'clickedInputLine' : 'inputLine'}
+              onChange={handleInput}
+              onFocus={() => setIsActiveEmailBorder(true)}
+              onBlur={() => setIsActiveEmailBorder(false)}
+            />
+          </div>
+          <div className="passwordInput">
+            <div className="loginInputMassage">Password *</div>
+            <input
+              name="password"
+              className={
+                isActivePasswordBorder ? 'clickedInputLine' : 'inputLine'
+              }
+              onChange={handleInput}
+              onFocus={() => setIsActivePasswordBorder(true)}
+              onBlur={() => setIsActivePasswordBorder(false)}
+            />
+          </div>
+          <span className="forgotPassword line">Forgot your password?</span>
+        </form>
+      </div>
+      <div className="loginBottom">
+        <button
+          className={handleButtonAction()}
```

Comment:
- `handleButtonAction` 함수 대신에 다음과 같이 바로 작성해볼 수 있을 것 같습니다.
```suggestion
          className={isInputValid ? 'loginButtonAction' : 'loginButtonInaction'}
```